docs/fundamentals/flow/flow-api.md

```diff
@@ -224,17 +224,26 @@ executors:
 
 ## Working with the Python Client
 The most convenient way to work with the `Flow` API is the Python Client. It enables you to send `Documents` to the `Flow` API in a number of different ways shown below:
+
+```{admonition} Caution
+:class: caution
+`Flow` provides a `.post()` method that follows the same interface as `client.post()`. Due to reasons of performance and
```

Comment:
These are not the reasons. However, when deploying in the Cloud the Flow interface will not be present